src/geo.js

```diff
@@ -5,5 +5,17 @@ module.exports = {
 	tile2lat: function(y,z) {
 		var n=Math.PI-2*Math.PI*y/Math.pow(2,z);
 		return (180/Math.PI*Math.atan(0.5*(Math.exp(n)-Math.exp(-n))));
-	}
+	},
+  geo2Webmercator: function(x_lon, y_lat){
+    if (Math.abs(x_lon) <= 180 && Math.abs(y_lat) < 90){
+      var num = x_lon * 0.017453292519943295;
```

Comment:
magic numbers don't help here, if you want to not use constants because it's faster, just leave some comments